hack/csi/manifests/vsphere/example-windows-pod.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
```

Comment:
This isn't vSphere specific, I'd suggest excluding this from the PR.

Reply:
Yeah this can be part of the CSI docs similar to [custom-workload.md](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/custom-workload.md)

Reply:
Removed